hack/update-btf

```diff
@@ -1,18 +1,28 @@
 #!/usr/bin/env bash
 set -euo pipefail
 
+BTFHUB_COMMIT=807c5c41894143513100ac58fa07e72df221baa1
 TEMP_DIR=$(mktemp -u)
 ROOT_DIR=$(git rev-parse --show-toplevel)
 BPF_OBJECT="$ROOT_DIR/build/recorder.bpf.o"
 BTF_DIR="$ROOT_DIR/internal/pkg/daemon/bpfrecorder/btf"
 
 rm -rf "$BTF_DIR"
-git clone --depth=1 https://github.com/aquasecurity/btfhub "$TEMP_DIR"
```

Comment:
Do you mind clarifying the removal of `--depth=1`? Not sure the benefit of removing it, and this may make such operation speedier.

Reply:
Yes the idea was to pin the BTF hub to a certain commit to not accidentally break the CI if it changes.

Reply:
That make sense, thank you.